### DISCUSSION TITLE
best way to change Text Input B when user change select input

### DISCUSSION THREAD

**Discussion by DVGY** [react-hook-form/react-hook-form#9435] [Q&A]
In was using `onChange` event handler to change the value in text Input B. 
```js
<Form>
  <SelectInput onChange={(e)=> setValue('textinputB' , "testing")} />
  <TextInput  name='testinputB' />
</Form>
```

How can I use the `useWatch` hook to achieve the same functionality. Is it advisable to use `useWatch` in this case` ?

**Comment by bluebill1049** [comment]
I think your approach is great.
